openlibrary/catalog/add_book/load_book.py

```diff
@@ -212,6 +212,14 @@ def build_query(rec):
                     raise InvalidLanguage(language)
             book['languages'] = [{'key': '/languages/' + language} for language in v]
             continue
+        if k == 'translated_from':
```

Comment:
This could be DRYed up with something like

```
if k in ('languages', 'translated_from'):
....
            book[k] = [
                {'key': '/languages/' + language} for language in v
            ]
```

(

Reply:
I have made this change.